docs/DeveloperGuide.md

```diff
@@ -1,38 +1,430 @@
-# Developer Guide
 
-## Acknowledgements
+---
+layout: page
```

Comment:
Are these two lines formatted correctly? The following is shown for me.
`layout: page title: Developer Guide —`